src/proto/grpc/testing/messages.proto

```diff
@@ -144,7 +140,7 @@ message StreamingOutputCallRequest {
   Payload payload = 3;
 
   // Whether to request the server to compress the response.
-  bool request_compressed_response = 4;
+  bool response_compressed = 6;
```

Comment:
note this field was introduced in this PR, so the tag renumbering is a no-issue
 Add a line note